src/makevoice/WindowsRegistry.cpp

```diff
@@ -0,0 +1,137 @@
+#include "stdafx.h"
+#include <strsafe.h>
+#include <tchar.h>
+#include <windows.h>
+#include <winnt.h>
+#include <winerror.h>
+#include <winreg.h>
+#include "WindowsRegistry.h"
+
+namespace WindowsRegistry
```

Comment:
Should we nest this namespace, and other namespaces we create, in a namespace like AssistiveTechAutomation, ATAutomation, W3C, etc?

Reply:
I deferred making this change for later if we want.